hazelcast/src/main/java/com/hazelcast/cache/impl/nearcache/impl/DefaultNearCache.java

```diff
@@ -109,6 +77,14 @@ protected ScheduledFuture scheduleExpirationTask(NearCacheExecutor nearCacheExec
         }
     }
 
+    private ScheduledFuture createExpirationTask() {
```

Comment:
I would call this 'initExpirationTask' since it does more than just creating; it also takes care of scheduling it on an executor. But... not that important.